cmd/bnsd/app/datamigration.go

```diff
@@ -44,4 +51,97 @@ func init() {
 			return nil
 		},
 	})
+
+	datamigration.MustRegister("rewrite username accounts", datamigration.Migration{
+		RequiredSigners: []weave.Address{governingBoard},
+		ChainIDs: []string{
+			"iov-mainnet",
+		},
+		Migrate: rewriteUsernameAccounts,
+	})
 }
+
+func rewriteUsernameAccounts(ctx context.Context, db weave.KVStore) error {
+	governingBoard, err := weave.ParseAddress("cond:gov/rule/0000000000000001")
+	if err != nil {
+		panic(err)
+	}
+
+	now, err := weave.BlockTime(ctx)
+	if err != nil {
+		return errors.Wrap(err, "block time")
+	}
+
+	iov := account.Domain{
+		Metadata:     &weave.Metadata{Schema: 1},
+		Domain:       "iov",
+		Admin:        governingBoard,
```

Comment:
@davepuchyr can you confirm who should be the admin of an IOV domain?

Reply:
I answered these question previously but don't see my comments.  Anyway, `governingBoard` should be the admin of the iov domain.